ggml/src/ggml-cpu/vec.h

```diff
@@ -351,6 +351,45 @@ inline static void ggml_vec_mad_f32_unroll(const int n, const int xs, const int
 #endif
 }
 
+inline static void ggml_vec_mad1_f32(const int n, float * y, const float s, const float b) {
+#if defined(GGML_USE_ACCELERATE)
+    vDSP_vsmsa(y, 1, &s, &b, y, 1, n);
+#elif defined(GGML_SIMD)
+    #if defined(__ARM_FEATURE_SVE)
+        // scalar ; TODO: Write SVE code
+        for (int i = 0; i < n; ++i) {
+            y[i] = y[i]*s + b;
+        }
+    #else
```

Comment:
`GGML_F32_STEP` doesn't seem to be defined on ARM SVE, so I leave the scalar impl here.

Reply:
Yes, it's ok for now. I'm having some doubts about these SVE branches - might end up removing them all together.